src/BlackSlope.Api/appsettings.docker.json

```diff
@@ -0,0 +1,9 @@
+{
+  "BlackSlope.Api": {
+    "Serilog": {
+      "WriteToFile": "true",
+      "WriteToConsole": "false"
+    },
+    "MoviesConnectionString": "data source=db,1433;initial catalog=movies;User Id=sa;Password=YourStrong!Passw0rd;MultipleActiveResultSets=True"

```

Comment:
Same comment about default passwords. Since this is a large PR, we can resolve this in a fast follow PR